### PR TITLE
希望加入一个简单获得绝对时间的方法, 包装了gettimeofday.

### PR DESCRIPTION
**Pull request by zii**
希望加入一个简单获得绝对时间的方法, 包装了gettimeofday.

### CODE REVIEW

File: cocos/base/ccUtils.cpp [hunk 1]
```diff
@@ -183,6 +183,14 @@ double atof(const char* str)
     
     return ::atof(buf);
 }
+
+double gettime()
+{
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
```
Comment (minggo) [inline cocos/base/ccUtils.cpp]: NULL -> nullptr

**Comment by minggo** [comment]
And why not declare it in ccUtils.h?

**Comment by zii** [comment]
谢谢指点, 我第一次用git, 多多包含.

**Comment by zii** [comment]
CI server crash
http://115.28.134.83:8000/job/cocos-2dx-pull-request-build/4481/console

**Comment by minggo** [comment]
@zii 
No problem, we have our own CI tools named jenkins.

**Comment by zii** [comment]
I see, but the api call failed at last.

**Pull request merged**